aten/src/ATen/native/native_functions.yaml

```diff
@@ -3018,6 +3018,14 @@
   variants: method
   device_guard: False
 
+- func: vander(Tensor x, *, bool increasing=False) -> Tensor
+  use_c10_dispatcher: full
+  supports_named_tensor: True
+
+- func: vander.n(Tensor x, int n, *, bool increasing=False) -> Tensor
```

Comment:
increasing can be specified positionally when n is specified, so you don't want the * here